lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
not sure if it'll get accepted, but I opened a PR against node-bindings to search this path that most native node modules use https://github.com/TooTallNate/node-bindings/pull/32